docs/index.html

```diff
@@ -23,14 +23,7 @@
   return response.json();
 }).then(function(data) {
   radar_visualization({
-    svg_id: "radar",
-    width: 1450,
-    height: 1000,
-    colors: {
-      background: "#fff",
-      grid: '#dddde0',
-      inactive: "#ddd"
-    },
+    tech_radar_homepage: "https://github.com/zalando/tech-radar",
```

Comment:
Would `repo_url` be more accurate here? (same as for [mkdocs](https://www.mkdocs.org/user-guide/configuration/#repo_url))